src/seedu/addressbook/data/person/address/Block.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person.address;
+
+/**
+ * Represents the block detail of the whole address
+ */
+public class Block {
+    public String block;
+
+    /**
+     * Initialises block with empty string
+     */
+    public Block() {
+        this.block = "";
+    }
+
+    /**
+     * Returns the block of the address
+     */
+    public String getBlock() {
+        return block;
+    }
+
+    /**
+     * Assigns string block with the block of the address
+     *
+     * @param block from the address
+     */
+    public void inputBlock(String block) {
```

Comment:
It is recommended that you follow the coding guidelines. Getter and setters should be getMember and setMember.